### ISSUE TITLE
1.5+ Breaks using Sentry 3.31 or higher

### ISSUE THREAD

**Issue by tibuprophen** [getsentry/sentry-xamarin#137]
### Environment

- MacOS 13.3.1 (a) (22E772610a)
- Xamarin Forms 5.0.0.2578

### Steps to Reproduce
(I'm using Rider, but the setup should be very similar to VS)
1. New Solution Xamarin Application (Cross Platform, Xamarin.Forms, .NET Standard, API 31)
2. Add Sentry.Xamarin.Forms 1.5.1 to all 3 Projects
3. Update the default Sentry Package to 3.31 or higher
4. Build App (with Release configuration)

Note: Using Sentry.Xamarin.Forms 1.4.6 will work with Sentry 3.31 or higher

### Expected Result
No Errors

### Actual Result
```
  Xamarin.Android.Legacy.targets(649, 5): [XALNK7000] Mono.Linker.MarkException: Error processing method: 'System.Void Xamarin.Forms.Platform.Android.ShellSearchView/ClipDrawableWrapper::.ctor(Android.Graphics.Drawables.Drawable)' in assembly: 'Xamarin.Forms.Platform.Android.dll' ---> Mono.Cecil.ResolutionException: Failed to resolve AndroidX.AppCompat.Graphics.Drawable.DrawableWrapper
  at Mono.Linker.Steps.MarkStep.HandleUnresolvedType (Mono.Cecil.TypeReference reference) [0x00013] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Steps.MarkStep.MarkType (Mono.Cecil.TypeReference reference) [0x0002c] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at MonoDroid.Tuner.MonoDroidMarkStep.MarkType (Mono.Cecil.TypeReference reference) [0x00000] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Steps.MarkStep.MarkType (Mono.Cecil.TypeReference reference) [0x00060] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at MonoDroid.Tuner.MonoDroidMarkStep.MarkType (Mono.Cecil.TypeReference reference) [0x00000] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Steps.MarkStep.ProcessMethod (Mono.Cecil.MethodDefinition method) [0x0001e] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Steps.MarkStep.ProcessQueue () [0x0001b] in <68c6268fbc45467eab046a8db2d5d932>:0 
   --- End of inner exception stack trace ---
  at Mono.Linker.Steps.MarkStep.ProcessQueue () [0x00047] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Steps.MarkStep.ProcessPrimaryQueue () [0x0000a] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Steps.MarkStep.Process () [0x000f6] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Steps.MarkStep.Process (Mono.Linker.LinkContext context) [0x0000d] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at MonoDroid.Tuner.MonoDroidMarkStep.Process (Mono.Linker.LinkContext context) [0x0000b] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Pipeline.ProcessStep (Mono.Linker.LinkContext context, Mono.Linker.Steps.IStep step) [0x0000d] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Mono.Linker.Pipeline.Process (Mono.Linker.LinkContext context) [0x0000f] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at MonoDroid.Tuner.Linker.Run (Mono.Linker.Pipeline pipeline, Mono.Linker.LinkContext context) [0x00000] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at MonoDroid.Tuner.Linker.Process (MonoDroid.Tuner.LinkerOptions options, Mono.Linker.ILogger logger, Mono.Linker.LinkContext& context) [0x00071] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Xamarin.Android.Tasks.LinkAssemblies.Execute (Java.Interop.Tools.Cecil.DirectoryAssemblyResolver res) [0x0018f] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Xamarin.Android.Tasks.LinkAssemblies.RunTask () [0x0001b] in <68c6268fbc45467eab046a8db2d5d932>:0 
  at Microsoft.Android.Build.Tasks.AndroidTask.Execute () [0x00000] in <ef4d16f91c3a4b36914e1b20639be304>:0 
``` 

Thank you for any help and generally your work. Please tell me if you need further information.

**Comment by lucas-zimerman** [comment]
Out of curiosity, Does it also break if you remove the Sentry dependency? I noticed another project having issues with those latest packages

**Comment by tibuprophen** [comment]
Hi
it seems more complex.
When I remove the Sentry dependency: 
- It breaks also on my given steps to reproduce. So it first seems no Sentry related bug, but...
- I also work on an App which breaks only if I update Sentry fron 1.4.x to 1.5.x. I can't upload the full source code of this. 

I'll try to upload an example very soon, where it breaks only by updating.

**Comment by michaelmairegger** [comment]
The latest working version of Sentry in combination with Sentry.Xamarin is 3.33.0.

So, the solution is to either downgrade **Sentry** to `3.33.0` or to downgrade **Sentry.Xamarin** to `1.4.6`

**Comment by lucas-zimerman** [comment]
Version 1.5.2 was released with support to Sentry 3.33.0. Is this still happening with you?

**Comment by tibuprophen** [comment]
Hello,

this Issue can be closed. Meanwhile, we've migrated our App to Maui. Since then the Issue is gone. 
Thank you for your Help!
